tests/main/interfaces-alsa/task.yaml

```diff
@@ -18,11 +18,12 @@ details: |
 
 prepare: |
     echo "Given a snap declaring a plug on the alsa interface is installed"
+    #shellcheck source=tests/lib/snaps.sh
     . $TESTSLIB/snaps.sh
     install_generic_consumer alsa
 
 restore: |
-    rm -f *.error /dev/snd/mysnd-dev
+    rm -f ./*.error /dev/snd/mysnd-dev
```

Comment:
I'm curious to know what was the warning here. Do you remember?

Reply:
```
In - line 1:
rm -f *.error /dev/snd/mysnd-dev
      ^-- SC2035: Use ./*glob* or -- *glob* so names with dashes won't become options.
```